### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by lucyJihyeon**
1. didn't style the navbar 
2. hero for the other pages 
3. background images 

223E77B3-7296-464A-897F-106BAC6377CE_1_102_o

### CODE REVIEW

File: client/src/components/Header/header.jsx [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
+  const [scrolled, setScrolled] = useState(false);
+  const { pathname } = useLocation();
+  console.log(pathname);
+
+  useEffect(() => {
+    // if pathnames are following, set the background color to be transparent 
+    // if (pathname === "/donate" || pathname === "/aboutus" || pathname === "/") {
+    //   setBackground(null); 
+    // } 
+    if (pathname === "/") {
```
Comment (lucyJihyeon) [inline client/src/components/Header/header.jsx:22]: Adding more Hero imgs here based on the path name

File: client/src/index.css [hunk 2]
```diff
@@ -26,13 +26,6 @@ a:hover {
   color: #535bf2;
 }
 
-body {
```
Comment (lucyJihyeon) [inline client/src/index.css:29]: this set up shifted the whole body element? and not filling up the whole screen when expanding/shrinking the screen size, so I deleted them for now

File: client/src/main.jsx [hunk 3]
```diff
@@ -39,7 +40,7 @@ const router = createBrowserRouter([
         element:<></>
       }, {
         path: 'browse',
-        element:<></>
+        element:<Browse />
```
Comment (lucyJihyeon) [inline client/src/main.jsx:43]: Added browse page to test out how the header background looks like

**Review by stephen-scheiman — APPROVED** [review]
Reviewed on 4/13

**Pull request merged**